Fakebook.Posts/Fakebook.Posts.RestApi/Services/CheckSpamService.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using Fakebook.Posts.Domain.Models;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Fakebook.Posts.Domain.Interfaces;
+
+
+namespace Fakebook.Posts.RestApi.Services
+{
+
+	public class CheckSpamService : ICheckSpamService
+	{
+		private readonly IPostsRepository _postRepository;
+		private readonly ITimeService _timeService;
+
+		public CheckSpamService(IPostsRepository postRepository, ITimeService timeService)
+		{
+			_postRepository = postRepository;
+			_timeService = timeService;
+		}
+
+		public async Task<bool> IsPostNotSpam(Post userPost)
+		{
+			if(userPost.Content == null)
+            {
+				return false;
+            }
```

Comment:
format this file, the whitespace is all over the place.

Reply:
for some reason, github is showing the brackets as far left when they are not.

Reply:
it's because you're mixing tabs and spaces, and github shows tabs with width 8